pkg/storage/chunk/chunk.go

```diff
@@ -47,11 +80,13 @@ type Chunk struct {
 
 	// The encoded version of the chunk, held so we don't need to re-encode it
 	encoded []byte
+
+	SecondaryIndexLabels SecondaryIndexLabels
```

Comment:
This is getting encoded to [json we generate here](https://github.com/grafana/loki-hackathon-2023-03-project-lili/blob/5829a1d8af6f2871c29b26a71ceeb46e149bebeb/pkg/storage/chunk/chunk.go#L289). We should ignore it while json encoding.
```suggestion
	SecondaryIndexLabels SecondaryIndexLabels `json:"-"`
```